server/app/views/api/ApiDocsView.java

```diff
@@ -169,32 +172,47 @@ private DivTag programDocsDiv(ProgramDefinition programDefinition) {
   }
 
   private DivTag questionDocsDiv(QuestionDefinition questionDefinition) {
-    DivTag divTag = div().withClasses("pl-4", "border", "border-gray-500", "rounded-lg");
+    DivTag divTag =
+        div()
+            .withClasses("pl-4", "border-b", "border-gray-300", "pt-2", "pb-2", "flex", "flex-col");
 
-    divTag.with(
+    DivTag topHalf = div();
```

Comment:
here and elsewhere:
Thank you for updating these names to be more clear! Could we go a little further and say what this is the top half of, like `questionCardHeader`, `questionCardBodyLeft` and `questionCardBodyRight`?

Reply:
Done.